README_libf3d.md

```diff
@@ -0,0 +1,215 @@
+# libf3d - A library to render 3D meshes
+
+By Michael Migliore and Mathieu Westphal.
+
+libf3d is a BSD-licensed C++ library to open and render 3D meshes. It is of course used by F3D.
+libf3d API is simple and easy to learn. Python bindings are provided through pybind11.
+
+# Getting Started
```

Comment:
We need to talk about CMake and using `find_package(f3d)` and the available targets.
Maybe an example of a basic CMake usage could be useful.

Reply:
added a section at the bottom